src/xml.h

```diff
@@ -164,16 +164,17 @@ void lyxml_ctx_free(struct lyxml_ctx *xmlctx);
 /**
  * @brief Compare values and their prefix mappings.
  *
- * @param[in] ctx libyang context.
+ * @param[in] ctx1 libyang context of First value.
```

Comment:
refer to the parameter: `@p value1`

Reply:
It was my initial idea as well but I let it be since `value1` does not have a context, strictly speaking. So the description will have to be a bit more verbose, something like `ctx1 libyang context for resolving prefixes in @p value1`.

Reply:
ok